cactusbot/api.py

```diff
@@ -13,7 +13,7 @@ def __init__(self, channel, **kwargs):
 
         self.channel = channel
 
-    async def add_command(self, name, response, *, modifiers={},
+    async def add_command(self, name, response, *, permissions={},
```

Comment:
Defaults should *never* be containers.
```python
In [1]: def f(x, a=[]):
   ...:     a.append(x)
   ...:     return a
   ...: 

In [2]: f(1)
Out[2]: [1]

In [3]: f(2)
Out[3]: [1, 2]

In [4]: f(3)
Out[4]: [1, 2, 3]
```
Instead, set it to `None`, and use `if permissions is None:` to reset it.